chapters/classes.tex

```diff
@@ -1544,11 +1544,18 @@ \subsection{Real Type}\label{real-type}
 \index{unbounded@\robustinline{unbounded}!attribute of \robustinline{Real}}%
 \index{stateSelect@\robustinline{stateSelect}!attribute of \robustinline{Real}}
 
+The following attributes shall be evaluable: \lstinline!quantity!, \lstinline!unit!, \lstinline!displayUnit!, \lstinline!fixed!, and \lstinline!stateSelect!.
+
 The \lstinline!unit! and \lstinline!displayUnit! attributes may be either the empty string or a string matching \lstinline[language=grammar]!unit-expression! in \cref{unit-expressions}.
 The meaning of the empty string depends on the context.
 For the input and output components of a function, the empty string allows different units to be used in different calls to the function.
 For a non-function component, the empty string allows the unit (or display unit) to be inferred by the tool.
 
+\begin{nonnormative}
+That \lstinline!displayUnit! is evaluable allows tools to verify that the default display unit is consistent with the \lstinline!unit!.
+Unlike the \lstinline!unit!, \lstinline!displayUnit! is just a default and unless given by a final modification, tools may allow overriding the default with other compatible display units in a translated model.
```

Comment:
```suggestion
Unlike the \lstinline!unit!, \lstinline!displayUnit! is just a default, tools may allow using other compatible display units for a translated model.
```
I don't see that it is necessary to mention `final` here and to me it it more that people _use_ another (display)Unit than _overriding_ the displayUnit of the model.

Consider
   `Real x(unit="s", final displayUnit="ms")=...;`
If I plot that and see that it has the value "2.345 ms" I still want to be able to convert that to "us" - and to me that's using a different displayUnit - not really overriding it; and to me it is best to not stress the details too much, but just be lose with "using a different displayUnit".

I understand that people rarely (if ever) make displayUnit final so in practice it is more about making the specification simpler.

Reply:
@henrikt-ma can you comment on this suggestion (preferably accepting it)?

Reply:
I'm sorry, I've bee to busy working on things related to unit handling lately.  Will look into it now.